plotters/src/element/pie.rs

```diff
@@ -31,6 +31,7 @@ pub struct Pie<'a, Coord, Label: Display> {
     label_style: TextStyle<'a>,
     label_offset: f64,
     percentage_style: Option<TextStyle<'a>>,
+    donut_hole: &'a f64, // radius of the hole in case of a donut chart
```

Comment:
Is there any reason for a lifetime here?

Reply:
Actually nope. Fixed by https://github.com/plotters-rs/plotters/pull/485/commits/fbfefd6f2c01c2077a06a8cace249f7ddf4cd6bf